package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "bluebird": "~2.9.25",
-    "fuel-auth": "~1.0.0",
+    "fuel-auth": "2.1.0",
```

Comment:
I've made a change in our app to give us some breathing room for this. I would rather it be upgraded as intended, if possible :)